data/strings/strings.txt

```diff
@@ -8461,6 +8461,99 @@
+    he = תמונות, חוות דעת, קישורי הזמנה מוצגים עבור מלונות שיש להם רישום באתר Kayak.com. בונוס ההפניה המתקבל עבור כל הזמנה דרך קישורים אלה מיועד לפיתוח של Organic Maps.
+    hi = जन हटल क Kayak.com पर सच ह, उनक तसवर, समकषए, बकग लक परदरशत कए जत ह। इन लक क मधयम स परतयक बकग क लए परपत रफरल बनस Organic Maps क वकस क लए जत ह।
+    hu = A Kayak.com-on szereplő szállodák esetében fényképek, értékelések, foglalási linkek jelennek meg. Az ezeken a linkeken keresztül történő minden egyes foglalásért kapott ajánlói bónusz az Organic Maps fejlesztését szolgálja.
+    id = Foto, ulasan, tautan pemesanan ditampilkan untuk hotel yang memiliki daftar di Kayak.com. Bonus rujukan yang diterima untuk setiap pemesanan melalui tautan ini digunakan untuk pengembangan Organic Maps.
+    it = Le foto, le recensioni e i link per le prenotazioni vengono visualizzati per gli hotel che hanno un'inserzione su Kayak.com. Il bonus di riferimento ricevuto per ogni prenotazione attraverso questi link è destinato allo sviluppo di Organic Maps.
+    ja = Kayak.comに掲載されているホテルの写真、レビュー、予約リンクが表示されます。これらのリンクを経由して予約された場合、紹介ボーナスはOrganic Mapsの開発に使用されます。
+    ko = Kayak.com에 숙소가 등록된 호텔의 사진, 후기, 예약 링크가 표시됩니다. 이러한 링크를 통해 예약할 때마다 받는 추천 보너스는 Organic Maps의 개발에 사용됩니다.
+    lt = Viešbučių, įtrauktų į Kayak.com sąrašą, nuotraukos, apžvalgos ir užsakymo nuorodos rodomos. Už kiekvieną per šias nuorodas atliktą užsakymą gauta premija skiriama "Organic Maps" plėtrai.
+    lv = Fotogrāfijas, atsauksmes, rezervēšanas saites tiek rādītas viesnīcām, kas ir iekļautas Kayak.com sarakstā. Par katru rezervāciju, kas veikta, izmantojot šīs saites, saņemtais ieteikuma bonuss tiek novirzīts Organic Maps attīstībai.
+    mr = Kayak.com वर सच असललय हटलसठ फट, पनरवलकन, बकग लकस परदरशत कल जतत. य लकसदवर परतयक बकगसठ मळलल रफरल बनस Organic Maps चय वकससठ जत.
+    nb = Bilder, anmeldelser og bestillingslenker vises for hoteller som har en oppføring på Kayak.com. Henvisningsbonusen som mottas for hver bestilling gjennom disse lenkene går til utvikling av Organic Maps.
+    nl = Foto's, beoordelingen en reserveringslinks worden weergegeven voor hotels die een vermelding op Kayak.com hebben. De verwijzingsbonus die voor elke boeking via deze links wordt ontvangen, gaat naar de ontwikkeling van Organic Maps.
+    pl = Zdjęcia, recenzje, linki do rezerwacji są wyświetlane dla hoteli, które mają wpis na Kayak.com. Premia za polecenie otrzymana za każdą rezerwację za pośrednictwem tych linków jest przeznaczona na rozwój Organic Maps.
+    pt = São apresentadas fotografias, críticas e links de reserva para os hotéis que têm uma listagem no Kayak.com. O bónus de referência recebido por cada reserva efectuada através destas ligações destina-se ao desenvolvimento do Organic Maps.
+    pt-BR = Fotos, avaliações e links de reserva são exibidos para os hotéis que estão listados no Kayak.com. O bônus de indicação recebido por cada reserva feita por meio desses links é destinado ao desenvolvimento do Organic Maps.
```

Comment:
pt = São apresentadas fotografias, críticas e ligações de reserva para os hotéis que estão listados no Kayak.com. O bónus de referência recebido por cada reserva efetuada através destas ligações destina-se ao desenvolvimento do Organic Maps.
    pt-BR = Fotos, avaliações e links de reserva são exibidos para os hotéis que estão listados no Kayak.com. O bônus de indicação recebido por cada reserva feita por meio desses links é destinado ao desenvolvimento do Organic Maps.